README.md

```diff
@@ -135,14 +135,17 @@ Supported properties in the `config.*.json` files:
 | `api-version`             | `v2`      | TranSMART API version. Only `v2` is supported. |
 | `app-url`                 |           | URL where the Glowing Bear is accessible for the user.|
 | `autosave-subject-sets`   | `false`   | Persist subject selection as subject set automatically. |
-| `export-data-view`        | `default` | Shape of the export (`default`, `surveyTable`). |
 | `show-observation-counts` | `true`    | |
 | `instant-counts-update-1` | `false`   | |
 | `instant-counts-update-2` | `false`   | |
 | `instant-counts-update-3` | `false`   | |
 | `authentication-service-type` | `oidc`  | Authentication service type (`oidc`, `transmart`) |
 | `oidc-server-url`         |           | E.g., `https://keycloak.example.com/auth/realms/{realm}/protocol/openid-connect` |
 | `oidc-client-id`          | `glowingbear-js` | |
+| `use-external-job`        | `false`   | Use an external tool that handles the export. Otherwise the dataView will be used. |
+| `export-data-view`        | `default` | Shape of the export (`default`, `surveyTable`).
+| `export-service-url`      |           | URL of the external tool that handles the export, if the `use-external-job` is set to `true`. |
+| `custom-export-job-name`  |           | Name of the export job that will be triggered in the external tool, if the `use-external-job` is set to `true`. |
```

Comment:
a bit difficult to understand what this custom-export-job-name is used for, the name will be triggered? doesn't sound like English, refactor the description

Reply:
`{ 
	"export-mode": {
		"mode": "packer",
	    "export-url": "http://foo.bar",
	    "data-view": "patient-dignosis"
	  }
}

=======

{ 
	"export-mode": {
		"mode": "transmart",
	    "data-view": "surveyTable"
	  }
}`

Reply:
@ewelinagr I edited the config json ⬆